auslib/db.py

```diff
@@ -2490,6 +2491,53 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None, dry
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction, dryrun=dryrun)
 
 
+class EmergencyShutoff(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('emergency_shutoff', metadata,
+                           Column('shutoff_id', Integer, primary_key=True, autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75), nullable=False),
+                           Column('updates_disabled', CompatibleBooleanColumn, default=False, nullable=False),
+                           Column('additional_notification_list', String(500)))
```

Comment:
I see nothing is actually using this yet - do you have an idea for how it will be implemented?

Reply:
People to be notified in addition to mailing list.
Oh! I forgot to include in mail list!
Any way, I can remove it, if it seems unnecessary for now.

Reply:
Yeah, I think that's best until there's actually code that will send e-mail when something happens. It's a nice idea though.